### PR TITLE
added windows support

### PR DESCRIPTION
**Pull request by skjerns**

### CODE REVIEW

File: pdf_compressor.py [hunk 1]
```diff
@@ -60,6 +61,14 @@ def compress(input_file_path, output_file_path, power=0):
     print("Done.")
 
 
+def get_ghostscript_path():
```
Comment (alexmahrou) [inline pdf_compressor.py:64]: Hi @skjerns I'm using python 2.7 so I don't have shutil.which. However I did have success in using your def to utilize the os module instead on my machine to return the correct path [as explained here](https://stackoverflow.com/questions/9877462/is-there-a-python-equivalent-to-the-which-command). I did have to add .exe to the variable though.

**Comment by theeko74** [comment]
Hi all,
Thanks for the interest in this little script.
Windows support is a good idea. Maybe we can first look in windows path and if not found looked in default path C:/Program Files/gs ?

**Comment by theeko74** [comment]
Thanks @alexmahrou for your pull request that has been integrated.
@runcioa I did not add your solution as probably we can solve that with instructions on how to add Ghoscript to path in windows. @alexmahrou, can you suggest something ?

**Comment by theeko74** [comment]
Sorry thanks @skjerns

**Pull request merged**